pkg/kudoctl/packages/install/package_test.go

```diff
@@ -81,30 +81,35 @@ func Test_InstallPackage(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		client := fake.NewSimpleClientset()
-		kc := kudo.NewClientFromK8s(client, kubefake.NewSimpleClientset())
+		tt := tt
 
-		fakeDiscovery, ok := client.Discovery().(*fakediscovery.FakeDiscovery)
-		if !ok {
-			t.Fatalf("couldn't convert Discovery() to *FakeDiscovery")
-		}
-		fakeDiscovery.FakedServerVersion = &version.Info{
-			GitVersion: "v1.16.0",
-		}
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
hmm why this change?

Reply:
Mostly because the test was failing and using the `t.Run(tt.name, func....` shows the separate cases nicely in IntelliJ when the tests are ran through the IDE